python/example_code/route53-recovery-cluster/routing_control_states.py

```diff
@@ -124,7 +127,8 @@ def toggle_routing_control_state(routing_control_arn, cluster_endpoints):
     parser = argparse.ArgumentParser()
     parser.add_argument('routing_control_arn', help="The ARN of the routing control.")
     parser.add_argument(
-        'cluster_endpoints', help="The list of endpoints for the cluster, in JSON format.")
+        'cluster_endpoints', help="A JSON file containing the list of endpoints for the cluster.")
     args = parser.parse_args()
-    toggle_routing_control_state(
-        args.routing_control_arn, json.loads(args.cluster_endpoints))
+    with open(args.cluster_endpoints) as endpoints_file:
+        cluster_endpoints = json.load(endpoints_file)
```

Comment:
Since cluster_endpoints should be an array, we would like to modify this to be

json.load(open(args.cluster_endpoints, 'r'))['Endpoints']

Reply:
Updated. I went with 'ClusterEndpoints' as the key, to match the name returned from the service.